API/app.test.js

```diff
@@ -108,5 +108,58 @@ describe('GET /api/reviews/:review_id', () => {
       });
 });
     
+describe('Get GET /api/reviews/:review_id/comments', () => {
+    it('responds with 200 and returns an array of comments for the given review_id', () => {
+        const review_id = 3
+        return request(app)
+            .get(`/api/reviews/${review_id}/comments`)
+            .expect(200)
+            .then(({body}) => {
+                expect(body.comments).toEqual(
+                    [{
+                        comment_id: 6,
+                        body: 'Not sure about dogs, but my cat likes to get involved with board games, the boxes are their particular favourite',
+                        review_id: 3,
+                        author: 'philippaclaire9',
+                        votes: 10,
+                        created_at: '2021-03-27T19:49:48.110Z'
+                      },
+                      {
+                        comment_id: 3,
+                        body: "I didn't know dogs could play games",
+                        review_id: 3,
+                        author: 'philippaclaire9',
+                        votes: 10,
+                        created_at: '2021-01-18T10:09:48.110Z'
+                      },
+                      {
+                        comment_id: 2,
+                        body: 'My dog loved this game too!',
+                        review_id: 3,
+                        author: 'mallionaire',
+                        votes: 13,
+                        created_at: '2021-01-18T10:09:05.410Z'
+                      }]
+                )
+                expect(body.comments).toBeSortedBy('created_at', {descending: true});
+            })
+    })
```

Comment:
We should have another 200 test here that returns an empty array if the article exists but has no comments